content/en/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: blog
+title: "Kubernetes 1.24: gRPC container probes in beta"
+date: 2022-05-13
+slug: grpc-probes-now-in-beta
+---
+
+**Author**: Sergey Kanzhelev (Google)
+
+
+With Kubernetes 1.24 the gRPC probes functionality entered beta and is available by default.
```

Comment:
I'm just a bit curious here since starting K8s v1.24, beta APIs will not be available by default. Since this is not an existing beta API, I'm not sure if I'm misunderstanding something with this sentence.
```suggestion
With Kubernetes 1.24 the gRPC probes functionality entered beta and is available by default.
```